src/azul/openapi/schema.py

```diff
@@ -117,6 +122,90 @@ def array(item: TYPE, *items: TYPE, **kwargs):
     return array_type(make_type(item), *map(make_type, items), **kwargs)
 
 
+def enum_type(items: Iterable) -> JSON:
```

Comment:
This should use varargs like other helpers. The name should be just `enum`.

@achave11 is working on this too. You guys should coordinate.

I like the implicit type determination but there should be a way to explicitly pass the type.